local.mk

```diff
@@ -1,5 +1,5 @@
 ifeq ($(MAKECMDGOALS), dist)
-  dist-files += $(shell git ls-files)
+  dist-files += $(shell cat dist-files)
```

Comment:
The problem with this is that it will break running `make dist` outside of a Nix build (since you have to run `postUnpack` to create `dist-files`). Not a big problem for me though.